gpjax/__init__.py

```diff
@@ -13,120 +13,37 @@
 # limitations under the License.
 # ==============================================================================
 from gpjax import (
+    base,
     decision_making,
+    gps,
     integrators,
-)
-from gpjax.base import (
-    Module,
-    param_field,
+    kernels,
+    likelihoods,
+    mean_functions,
+    objectives,
+    variational_families,
 )
 from gpjax.citation import cite
 from gpjax.dataset import Dataset
 from gpjax.fit import fit
-from gpjax.gps import (
-    Prior,
-    construct_posterior,
-)
-from gpjax.kernels import (
-    RBF,
-    RFF,
-    AbstractKernel,
-    BasisFunctionComputation,
-    CatKernel,
-    ConstantDiagonalKernelComputation,
-    DenseKernelComputation,
-    DiagonalKernelComputation,
-    EigenKernelComputation,
-    GraphKernel,
-    Linear,
-    Matern12,
-    Matern32,
-    Matern52,
-    Periodic,
-    Polynomial,
-    PoweredExponential,
-    ProductKernel,
-    RationalQuadratic,
-    SumKernel,
-    White,
-)
-from gpjax.likelihoods import (
-    Bernoulli,
-    Gaussian,
-    Poisson,
-)
-from gpjax.mean_functions import (
-    Constant,
-    Zero,
-)
-from gpjax.objectives import (
-    ELBO,
-    CollapsedELBO,
-    ConjugateMLL,
-    LogPosteriorDensity,
-    NonConjugateMLL,
-)
-from gpjax.variational_families import (
-    CollapsedVariationalGaussian,
-    ExpectationVariationalGaussian,
-    NaturalVariationalGaussian,
-    VariationalGaussian,
-    WhitenedVariationalGaussian,
-)
 
 __license__ = "MIT"
 __description__ = "Didactic Gaussian processes in JAX"
 __url__ = "https://github.com/JaxGaussianProcesses/GPJax"
 __contributors__ = "https://github.com/JaxGaussianProcesses/GPJax/graphs/contributors"
-__version__ = "0.7.1"
+__version__ = "0.7.2"
```

Comment:
Technically this is a breaking change (`gpx.Matern52` will no longer work as before), so should be
```suggestion
__version__ = "0.8.0"
```
if you want to follow semantic versioning (which I would be strongly in favour of).

Reply:
Agreed